public/app/system.conf.js

```diff
@@ -1,6 +1,15 @@
+function getBaseURL() {
+  "use strict";
+  /* in the case of system.build, window is undefined
+     We should still pull the plugins from the CDN*/
+  if(typeof(window) === "undefined") {
+    return "public";
+  }
+  return window.grafanaBootData.settings.appStaticUrl || "public";
+}
 System.config({
   defaultJSExtenions: true,
-  baseURL: 'public',
+  baseURL: getBaseURL(),
```

Comment:
I am not sure if this change is necessary.
Do the plugin system uses this? in this case this is necessary. If its only the config for the boot.js generation, then I will remove the change.
Please advise.